apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx

```diff
@@ -1,4 +1,3 @@
-import { AspectRatio } from "@/components/ui/aspect-ratio";
```

Comment:
@jnsdls I didn't know there was another AspectRatio so I mistakenly removed this one. And now the lint is telling me due to this change, `@radix-ui/react-aspect-ratio` is no longer being used.

So, should we remove said dependency, or use in as the replacement for all the Chakra-ui's AspectRatio?

Reply:
(I have a feeling you would want to remove the dep)

Reply:
@MananTank had thoughts on this last time it came up, will have him make the decision on what we do here, but either way it's either we use css or we use the component, we have to pick one of the options, not both

Reply:
We can replace with it with CSS/tailwind class here but my only concern is that it's not a an exact replacement, (I've seen this when I refactored the FileInput component - I had to make some other changes to make the aspect-ratio work exactly as chakra version ). 

Lets verify that all changes are working as expected and we are not accidentally breaking the layout

Reply:
@MananTank yeah it's not working exactly the same you're right. I had to make few minor changes to re-align the inner content (re-center, more like).

I guess that means we stick with `@radix-ui/react-aspect-ratio`

Reply:
Let's replace the instances where it works with just a css class and make changes in other places as needed and remove AspectRatio component

Reply:
Tested this PR. should be gud